examples/example-vite-react-sdk/src/useSystemCalls.ts

```
@@ -0,0 +1,62 @@
+        const transactionId = uuidv4();
+
+        // The value to update the Moves model with
+        const remainingMoves = 100;
+
+        // Apply an optimistic update to the state
+        // this uses immer drafts to update the state
+        state.applyOptimisticUpdate(
+            transactionId,
+            (draft) =>
+                (draft.entities[entityId].models.dojo_starter.Moves!.remaining =
+                    remainingMoves)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)